be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -130,12 +131,22 @@ class OrcRowReaderFilter : public orc::RowReaderFilter {
     // 2. and check if range.start <= `offset`
     std::map<uint64_t, uint64_t> _scan_ranges;
     OrcScannerAdapter* _adapter;
+    int64_t _writer_tzoffset_in_seconds;
```

Comment:
why not give this field a default value?

Reply:
No need to set default value, because this field will be updated during opening strip in ORC file. But It's okay to set default value

```
void OrcRowReaderFilter::setWriterTimezone(const std::string& tz) {
    cctz::time_zone writer_tzinfo;
    if (tz == "" || !TimezoneUtils::find_cctz_time_zone(tz, writer_tzinfo)) {
        _writer_tzoffset_in_seconds = _adapter->tzoffset_in_seconds();
    } else {
        _writer_tzoffset_in_seconds = TimezoneUtils::to_utc_offset(writer_tzinfo);
    }
}
```